python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -766,33 +800,11 @@ async def correction_loop(self):
                     # filter/disperser changes, then apply if needed
                     self.log.debug(f"self.corrections['atspectrograph']"
                                    f" is {self.corrections['atspectrograph']}")
-                    self.log.debug(f"self.focus_offset_yet_to_be_applied "
-                                   f"is {self.focus_offset_yet_to_be_applied}")
-                    # check that offset is not just numerical noise. In our case anything smaller than
-                    # the correction tolerance won't be applied to the hexapod anyways, so any offset
-                    # 10x smaller than that is surely noise.
-                    if self.corrections['atspectrograph'] and (
-                            abs(self.focus_offset_yet_to_be_applied) >= abs(
-                            self.correction_tolerance['z'] / 10)):
-                        self.log.info(
-                            f'Applying focus offset of {self.focus_offset_yet_to_be_applied} '
-                            f'in correction loop due to filter '
-                            f'and/or disperser changes.')
-                        # add the offset, then reset the value
-                        # using subtraction here to avoid a possible race condition
-                        # note that self.focus_offset_yet_to_be_applied is a value not an object
-                        # so no deepcopy is required
-                        _offset_value = self.focus_offset_yet_to_be_applied
-                        self.model.add_offset("z", _offset_value)
-                        self.focus_offset_yet_to_be_applied -= _offset_value
-                        # publish events with new offsets
-                        self.evt_correctionOffsets.set_put(**self.model.offset,
-                                                           force_output=True)
-                        # Do accounting to republish total offset and others that were set
-                        # in the callbacks
-                        self.focus_offset_per_category['total'] = self.model.offset['z']
-                        self.evt_focusOffsetSummary.set_put(**self.focus_offset_per_category,
-                                                            force_output=True)
+                    self.log.debug(f"self.focusOffsetYetToBeApplied "
```

Comment:
remove "f" from first line of string... it is ugly but the "correct" way is to do something like:

```
self.log.debug("self.focusOffsetYetToBeApplied "
                          f"is {self.focusOffsetYetToBeApplied}")
```

I think newer versions of flake-8 complain about that.

Reply:
Ok, removed a bunch of these types of occurrences throughout the code and in the test_csc. Not promising I got all of them though...